unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/ParcelScene.cs

```diff
@@ -348,8 +337,8 @@ public DecentralandEntity CreateEntity(string id)
 
             newEntity.OnCleanupEvent += po.OnCleanup;
 
-            if (useBoundariesChecker || SceneController.i.isDebugMode)
-                newEntity.OnShapeUpdated += boundariesChecker.EvaluateEntityPosition;
+            if (SceneController.i.useBoundariesChecker)
+                newEntity.OnShapeUpdated += SceneController.i.boundariesChecker.AddEntity;
```

Comment:
If an entity is removed we should remove it from this queue too or we might get weird errors.